reader.js

```diff
@@ -7,15 +9,156 @@ const CLOSE_TIMEOUT = 10000;
 
 let WSS;
 
-function init(wss) {
+function init(server) {
     console.log('Initializing nasonov-reader');
+    const wss = new WebSocket.Server({server});
 
     WSS = wss;
     wss.on('connection', function connected(ws, req) {});
 
+    server.on('request', (request, response) => {connectToInflux(request, response)});
+
     connectToWriter();
 }
 
+function connectToInflux(request, response) {
```

Comment:
We want the names of functions to explain what we do. How about something more like `respondToHTTPRequest`?